### PR TITLE
Add "Python Package" YAML template

### PR DESCRIPTION
**Pull request by brcrista**
YAML template for building a Python package and testing on multiple Python versions.

Testing: built the Python Canary Build repo with this YAML template

### CODE REVIEW

**Comment by brcrista** [comment]
@DavidStaheli should we put the queue in the YAML?  It seems like if you specify queue in the YAML, you can't change it when you queue the build.  But there also seems to be a bug in the UI where it reports this incorrectly in the build summary:

image

Also, it seems like the YAML schema demands you specify the queue separately for each phase.

**Comment by DavidStaheli** [comment]
I like leaving the queue in the YAML.  The designer UI will be overridden by the YAML value (soon if not already).  I'd like to look into templating [Phase Reuse - coming soon](https://jitaccess.security.core.windows.net/WorkFlowTempAccess.aspx?View=Submit&ResourceType=Elevated Security Group(AAD)&TargetSystem=VSTS-CSSAdminTool&Scenario=JIT-CSSAdminTool-HighPrivileges&AccessLevel=Member&WorkItemID=1924273&WorkItemSource=TFS:RD) to see if we can define the phase at the top of the YAML, and use it 3 times below with different queue names.

File: templates/ci/python-package.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - script: python -m pip install --upgrade pip && pip install wheel && python setup.py bdist_wheel
+    displayName: Build Wheel
+  - task: PublishBuildArtifacts@1
+    displayName: 'Publish artifacts'
+    inputs:
+      pathToPublish: 'dist'
+      artifactName: 'artifact'
+      artifactType: 'container'
+  # - task: PyPIPublisher@0
+  #   inputs:
```
[the remark this answers is not in the document]
Comment (brcrista) [reply templates/ci/python-package.yml]: good catch

[the remark this answers is not in the document]
Comment (brcrista) [reply templates/ci/python-package.yml]: I wish "show YAML" would just show you all the inputs

[the remark this answers is not in the document]
Comment (madhurig) [reply templates/ci/python-package.yml]: It should at least show the "required" inputs :)

**Pull request merged**